sdk/communication/Azure.Communication.Identity/src/autorest.md

```diff
@@ -7,6 +7,6 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/3e973fa7cd38a218772b1f5960d60b21da365239/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json
+    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5b19c6e69cd2bb9dbe4e5c1237b2c5a175d90ca5/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json
```

Comment:
Which commit is the one we should use officially?

JS is using yet another commit it seems.

Reply:
`5b19c6e69cd2bb9dbe4e5c1237b2c5a175d90ca5` seems to be the latest commit that got merged 2 days ago